build.gradle

```diff
@@ -24,11 +24,11 @@ targetCompatibility = JavaVersion.VERSION_11
 
 ext {
     libs = [
-        'spring': '5.3.2',
-        'jackson': '2.12.2',
-        'lombok': '1.18.18',
+        'spring': '5.3.6',
```

Comment:
Also taking the opportunity to bump some minor version dependencies, to save a separate release for it.